devices/ubisys.js

```diff
@@ -925,4 +925,15 @@ module.exports = [
         },
         ota: ota.ubisys,
     },
+    {
+        zigbeeModel: ['R0 (5501)'],
+        model: 'R0',
+        vendor: 'ubisys',
+        description: 'Zigbee Router R0',
+        supports: 'Router',
+        fromZigbee: [],
+        toZigbee: [],
+        exposes: [],
+        ota: ota.ubisys,
+    }
```

Comment:
```suggestion
    },
```

Missing trailing ,

Reply:
wow, you answered and fixed my mistake faster than I could understand the output of the lint checker and make a new commit ;-)
I was very unsure about the configure or meta section you added on other ubisys devices. m2z gives an error when I click reconfigure on the R0. If you have some ideas, I would be happy to try them with my R0. Unless you think they are unnecessary.

Reply:
There is nothing needed in the meta block, also this device won't need a configuration function either, as there is nothing to configure for it.

Reply:
Allright. Thanks a lot :) 
I will try to update the devices description in z2m and link the PR here.

Reply:
If I have your attention already, maybe I can ask you an unrelated question about LQI reportet in Homeassistant. I can see the LQI of each device on the Devices and Map pages of m2z but in HA (and over MQTT) only the LQI of the routers are reportet, not the end devices. I have found conflicting posts online (although I don't have the links at hand), one where Koenkk explains that this is how zigbee works, another where someone ammended the config of a coordinator to forward endpoint LQIs - at least that is how I (mis)understood. Do you know  which one is correct?

Reply:
I've never really dug into LQI values at all. I vaguely remember something about z2m showing what the device reported.